xyz-hub-service/src/main/java/com/here/xyz/httpconnector/config/JDBCClients.java

```diff
@@ -281,7 +280,7 @@ public static Future<RDSStatus> getRDSStatus(String clientId) {
                         JSONObject avg5MinRDSMetrics = CService.jobCWClient.getAvg5MinRDSMetrics(CService.rdsLookupDatabaseIdentifier.get(clientId));
                         p.complete(new RDSStatus(clientId, avg5MinRDSMetrics, runningQueryStatistics));
                     }).onFailure(f -> {
-                        logger.warn("Cant get RDS-Resources {}",f);
+                        logger.warn("Cant get RDS-Resources {}", f.getMessage());
```

Comment:
That way the stack-trace will be missing in the logs. Please replace f.getMessage() by `f` and also remove the second {} in the message text. Otherwise the effect would be the same.